tests/lib/cli.js

```diff
@@ -171,12 +219,15 @@ describe("cli", function() {
 
     describe("when calling execute more than once", function() {
         it("should not print the results from previous execution", function() {
-            cli.execute("--no-ignore tests/fixtures/missing-semicolon.js");
+            var configPath = getFixturePath("configurations", "semi-error.json");
+            var filePath = getFixturePath("missing-semicolon.js");
+
+            cli.execute("--no-ignore --config " + configPath + " " + filePath);
             assert.isTrue(console.log.called, "Log should have been called.");
 
             console.log.reset();
 
-            cli.execute("--no-ignore tests/fixtures/passing.js");
+            cli.execute("--no-ignore --config " + configPath + " " + filePath);
```

Comment:
You are passing in wrong file thats why test is failing.
You have to use `passing.js` file here.